intel-retail/automated-self-checkout
# feat: add yolov5s + efficientnet-b0 cAPI Profile

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #359 

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable
make build-capi_yolov5_ensemble
make clean-all
make run-camera-simulator
run the pipeline: PIPELINE_PROFILE="capi_yolov5_ensemble" RENDER_MODE=1 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0 --workload ovms

upon successful you should see FPS value, and result video playback with bounding boxes.

make run-smoke-tests

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
